docs/app/pages/components/components-sections/file-upload/file-upload/file-upload.component.html

```diff
@@ -63,9 +63,13 @@
     files into an area on the page.
 </p>
 
-<p>
-    Refer to <a href="https://valor-software.com/ng2-file-upload/">ng2-file-upload</a> for complete documentation on this component.
-</p>
+<blockquote>
+    <p><strong>Note</strong>: If you are using this component in your application, you will need to install <strong>ng2-file-upload</strong> manually.</p>
+    <p>
+        Refer to <a href="https://valor-software.com/ng2-file-upload/" target="_blank">ng2-file-upload</a> for complete documentation on this component.
```

Comment:
```suggestion
        Refer to <a href="https://valor-software.com/ng2-file-upload/" target="_blank" rel="noopener">ng2-file-upload</a> for complete documentation on this component.
```
https://web.dev/external-anchors-use-rel-noopener/